src/detect_breakends/junction_detection.cpp

```diff
@@ -1,5 +1,14 @@
 #include "detect_breakends/junction_detection.hpp"
 
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/io/alignment_file/input.hpp>   // SAM/BAM support
+#include <seqan3/io/sequence_file/output.hpp>   // FASTA support
+
+#include "detect_breakends/bam_functions.hpp"   // for hasFlag* functions
+#include "junction.hpp"                         // for class Jnction
+#include "cluster.hpp"                          // for class Cluster
+#include "detect_breakends/aligned_segment.hpp" // for struct AlignedSegment
```

Comment:
Does iGenVar require alphabetical order?

Reply:
Yes we try :) thanks